css/default.css

```diff
@@ -1258,7 +1267,8 @@ div#defaultsidebar .mysidebar .sidebarbuttoncontainer {
 }
 
 .highlightlink {
-  background-color: var(--body-background) !important; /* Make sure background is white */
+  background-color: var(--body-background) !important;
```

Comment:
Dto., please keep the comment behind the declaration as it’s related to it. (On the next line, it looks like it relates to the “color” declaration.)